srcpkgs/compton/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'compton'
 pkgname=compton
-version=0.0.20180521
+version=0.0.20180817
```

Comment:
Wrong date.
This [commit](https://github.com/yshui/compton/commit/cac8094ce12cd40706fb48f9ab35354d9ee7c48f) is dated Aug 11, 2018.